examples/Container/Frontend/Balancer/ConfigurableResolverFactory.cs

```diff
@@ -101,6 +103,11 @@ private void RaiseResult(ResolverResult result)
                         // DNS results change order between refreshes.
                         // Explicitly order by host to keep result order consistent.
                         var orderedAddresses = result.Addresses.OrderBy(a => a.EndPoint.Host).ToList();
+                        // Remove host override from addresses so the destination IP address is available.
```

Comment:
```suggestion
                        // Remove host override from addresses so the destination IP address is available.
                        // The sample does this because the server returns the IP address to the client.
                        // This makes it clear that gRPC calls are balanced between pods.
```